permissions/plugin-artifactory.yml

```diff
@@ -1,6 +1,8 @@
 ---
 name: "artifactory"
 github: "jenkinsci/artifactory-plugin"
+issues:
+- jira: '15689' # artifactory-plugin
```

Comment:
FTR they handler only submissions to their own tracker.

Reply:
🤷